src/System.Net.Requests/src/System/Net/HttpVersion.cs

```diff
@@ -0,0 +1,28 @@
+//------------------------------------------------------------------------------
+// <copyright file="HttpVersion.cs" company="Microsoft">
+//     Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>                                                                
+//------------------------------------------------------------------------------
+
+namespace System.Net {
+
+    /// <devdoc>
+    ///    <para>
+    ///       Defines the HTTP version number supported by the <see cref='System.Net.HttpWebRequest'/> and
+    ///    <see cref='System.Net.HttpWebResponse'/> classes.
+    ///    </para>
+    /// </devdoc>
+    public class HttpVersion {
```

Comment:
@davidsh, I'm curious: is `System.Net.Requests` the right long-term place for this now-public type or is there a "lower-level" assembly that would be a better fit? There is already an internal copy used by `System.Net.Http` and `System.Net.Http.WinHttpHandler` here: https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/HttpVersion.cs